Sources/Theme.php

```diff
@@ -2745,25 +2776,9 @@ protected function loadVariant(): void
 				Utils::$context['theme_variant'] = !empty($this->settings['default_variant']) && in_array($this->settings['default_variant'], $this->settings['theme_variants']) ? $this->settings['default_variant'] : $this->settings['theme_variants'][0];
 			}
 
-			if (!empty($this->settings['has_dark_mode'])) {
-				if (Utils::$context['theme_variant'] == '' || Utils::$context['theme_variant'] == 'system') {
-					foreach (['light',  'dark'] as $var) {
-						self::loadCSSFile('index_' . $var . '.css', ['order_pos' => 301, 'attributes' => ['media' => '(prefers-color-scheme: ' . $var . ')']], 'smf_index_' . $var);
-					}
-				} else {
-					self::loadCSSFile('index_' . Utils::$context['theme_variant'] . '.css', ['order_pos' => 301], 'smf_index_' . Utils::$context['theme_variant']);
-				}
-			} elseif (Utils::$context['theme_variant'] != '') {
-				self::loadCSSFile('index_' . Utils::$context['theme_variant'] . '.css', ['order_pos' => 300], 'smf_index_' . Utils::$context['theme_variant']);
-
-				if (Utils::$context['right_to_left']) {
-					self::loadCSSFile('rtl_' . Utils::$context['theme_variant'] . '.css', ['order_pos' => 4200], 'smf_rtl_' . Utils::$context['theme_variant']);
-				}
+			if (!empty(Utils::$context['theme_variant']) && Utils::$context['theme_variant'] !== 'default') {
```

Comment:
I think this may subvert expectations if a theme author decides to name a variant `default`

```suggestion
			if (!empty(Utils::$context['theme_variant'])) {
```

Reply:
“default” is not expected anymore besides just being a “dummy” value to come back to the main colors. It does bring up a point of someone adding a “Default” variant, but it would be different because event the text string is not case sensitive.
variant_default vs variant_Default? I suppose it can be allowed, doesn’t make a difference.